R/getCells.R

```diff
@@ -16,23 +16,21 @@
 #' \code{\link{setNames}}, \code{\link{getCPR}}, \code{\link{read.magpie}},
 #' \code{\link{write.magpie}}, \code{"\linkS4class{magpie}"}
 #' @examples
-#' 
-#'  a <- as.magpie(1)
+#'
+#' a <- as.magpie(1)
 #'  getCells(a)
-#'  setCells(a,"AFR")
-#' 
+#'  setCells(a, "AFR")
 #' @export
 getCells <- function(x) {
   return(dimnames(x)[[1]])
 }
 
 #' @describeIn getCells set cell names
 #' @export
-"getCells<-" <- function(x,value) {
-  if(length(value)!=ncells(x)) stop("Wrong number of cell names supplied!")
-  if(ncells(x)==0) return(x)
-  if(is.null(value)) stop("Setting cell names to NULL is not allowed!")
-  if(length(value)==1) value <- list(value)
+"getCells<-" <- function(x, value) { #nolint
```

Comment:
```suggestion
`getCells<-` <- function(x, value) { #nolint
```
Assigning something to a string looks super weird to me. Also, you can call the function using
```
`getCells<-`(...)
```
but not with `"getCells<-"()`

Reply:
This also might mean we don't need the `#nolint` anymore